{"name": "test_complete_task_with_failure", "status": "passed", "steps": [{"name": "Set up Task for failure", "status": "passed", "start": 1733724310333, "stop": 1733724310524}, {"name": "Complete the task with failure", "status": "passed", "start": 1733724310524, "stop": 1733724310651}, {"name": "Verify that a log is created for the failed task", "status": "passed", "start": 1733724310651, "stop": 1733724310683}, {"name": "Verify that an alert is created for the failed task", "status": "passed", "start": 1733724310683, "stop": 1733724310714}, {"name": "Verify that billing is recorded for the failed task", "status": "passed", "start": 1733724310714, "stop": 1733724310746}], "start": 1733724310333, "stop": 1733724310747, "uuid": "332906e7-a988-46da-bb6d-363e7b0d17d1", "historyId": "4821618ed2bb1494de2b360ed1839571", "testCaseId": "4821618ed2bb1494de2b360ed1839571", "fullName": "user.tests.test_big_data#test_complete_task_with_failure", "labels": [{"name": "story", "value": "Complete Task with Failure"}, {"name": "feature", "value": "Task Management"}, {"name": "tag", "value": "django_db"}, {"name": "parentSuite", "value": "user.tests"}, {"name": "suite", "value": "test_big_data"}, {"name": "host", "value": "SHPatsnapB0008"}, {"name": "thread", "value": "49308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "user.tests.test_big_data"}]}